client/tos/modal/style.scss

```diff
@@ -28,4 +28,8 @@
 			height: auto;
 		}
 	}
+
+	.woocommerce-payments__tos-error {
+		margin: 0 0 1em;
```

Comment:
Minor thing but `em` is not used that often in our codebase, `rem` or `px` seem to be preferred.

Reply:
The style is removed because of the usage of the notices API.